code/go/0chain.net/smartcontract/storagesc/block_reward.go

```diff
@@ -35,32 +39,41 @@ func (ssc *StorageSmartContract) blobberBlockRewards(
 
 	conf, err := ssc.getConfig(balances, true)
 	if err != nil {
-		return common.NewError("blobber_block_rewards_failed",
+		return "", common.NewError("blobber_block_rewards_failed",
```

Comment:
Seems we are returning empty string in all cases, perhaps we don't need it or return something as transaction output?